src/components/lookup.js

```diff
@@ -129,4 +150,62 @@ export const Lookup = (props) => {
                 </Wrapper>
         
     
+}
+
+export const SearchedHolos = (props) => {
+  const [userHolos, setUserHolos] = useState([])
+  const [loading, setLoading] = useState(true)
+
+  async function getHolos() {
+    console.log('Provier...', props.provider)
+    setLoading(true)
+
+    // Get all addresses with name/bio
+    console.log('Entered getHolos in lookup.js')
+    const wtfBiosAddr = contractAddressesNew['production']['WTFBios']['polygon']
+    const nameAndBioContract = new ethers.Contract(wtfBiosAddr, wtfBiosABI, props.provider)
+    console.log('Calling nameAndBioContract.getRegisteredAddresses() in lookup.js')
+    const addrsWithNameOrBio = await nameAndBioContract.getRegisteredAddresses()
+    // const addrsWithNameOrBio = ['0xcaFe2eF59688187EE312C8aca10CEB798338f7e3']
+  
+    console.log('addrsWithNameOrBio...', addrsWithNameOrBio)
+
+    // Get all creds of every account with a name/bio that includes search string
+    let allHolos = []
+    for (const address of addrsWithNameOrBio) {
+      console.log('Getting holo for address...', address)
+      const holoData = await wtf.getHolo(address)
+      let name = holoData[props.desiredChain]['name']
+      let bio = holoData[props.desiredChain]['bio']
+      if (props.searchStr.includes(name) || props.searchStr.includes(bio)) {
+        let creds = holoData[props.desiredChain]['creds']
+        let holoTemp = {
+          'name': name,
+          'bio': bio,
+          'twitter': creds['twitter'],
+          'google': creds['google'],
+          'github': creds['github'],
+          'orcid': creds['orcid'] || '0000-6969-6969'
```

Comment:
lmao of all the default creds you kept this one

i'll merge it but our audience may not be as cool as we are so i'll change it after